src/Mod/TechDraw/Gui/QGCustomText.cpp

```diff
@@ -185,6 +191,32 @@ void QGCustomText::paint ( QPainter * painter, const QStyleOptionGraphicsItem *
     QGraphicsTextItem::paint (painter, &myOption, widget);
 }
 
+QRectF QGCustomText::boundingRect() const
```

Comment:
Pure aesthetics, but this feels like a lot of vertical space for a very simple function: it would be (slightly) more compact in the form of an if-elseif-else.
```suggestion
QRectF QGCustomText::boundingRect() const
{
    if (toPlainText().isEmpty()) {
        return QRectF();
    } else if (tightBounding) {
        return tightBoundingRect();
    } else {
        return QGraphicsTextItem::boundingRect();
    }
}
```